src/pages/transactions/[id].js

```diff
@@ -1,42 +1,43 @@
 import { useRouter } from "next/router";
 import Link from "next/link";
 import Modal from "@/src/components/Modal";
-import ChevronLeft from "@/src/assets/icons/chevron_left.svg";
-import Button from "@/src/components/styles/Button";
-import SVGIcon from "@/src/components/SVGIcon";
+import { useState } from "react";
 
 
 export default function TransactionDetails({
-                                             initialData,
+                                             transactions,
                                              onDelete,
-                                             mode,
-                                             onMode,
                                            }) {
   const router = useRouter();
   const { id } = router.query;
+  const [modal, setModal] = useState(false);
 
   // Find transaction by its ID
-  const transaction = initialData.find((transaction) => transaction.id === id);
+  const transaction = transactions.find((transaction) => transaction.id === id);
 
   if (!transaction) {
     return <p>Transaction not found</p>;
   }
+  console.log(transaction);
+  const displayAmount = `${transaction.transactionType === "Expense" && "-"}${transaction?.amount.toFixed(2)}`;
```

Comment:
When viewing a transaction's detail, a 'false' appears in front of the amount when the transaction type is'Income' (see attached screenshot). I guess this originates here. You can also delete the console log. 
<img width="166" alt="dagobert_screen" src="https://github.com/miToe/dagobert/assets/164894965/74e10f8e-80b0-4959-8537-ee051ecef15a">

Reply:
This happens after you switch the branch and dont clear the localstorage. I deleted the console log.